apps/sensenet/src/components/context-menu/content-context-menu.tsx

```diff
@@ -44,6 +44,13 @@ export const ContentContextMenu: React.FunctionComponent<ContentContextMenuProps
       }
       const contentActions = contentFromCallback.Actions.filter((action) => !action.Forbidden)
 
+      if (contentActions.findIndex((action) => action.Name === 'Browse' && props.content.IsFile)) {
```

Comment:
I would rather use `array.some` here. findIndex can return with 0 which is falsy but a valid index number.

Reply:
Changed it `some` and changing it also highlighted another bug with the props.content so it is also changed to contentFromCallback :)